src/climate_learn/data/climate_dataset/era5/era5.py

```diff
@@ -92,7 +92,7 @@ def setup_constants(self, data_dir: str) -> None:
             all_constants = xr.open_mfdataset(ps, combine="by_coords")
             for name in self.constants:
                 self.constants_data[name] = torch.tensor(
-                    all_constants[NAME_TO_VAR[name]].values
+                    all_constants[NAME_TO_VAR[name]].values.astype(numpy.float32)
```

Comment:
For the sake of code consistency, IMO we should `import numpy as np` so this should be `.astype(np.float32)`. Same for the below change.

Reply:
Sure.

Reply:
Addressed.